docs/articles/rag-application-communication-system.md

```diff
@@ -0,0 +1,164 @@
+
+While this is much less used, the exact same approach (can be used for embedding models)[https://huggingface.co/blog/davanstrien/synthetic-similarity-datasets]. Synthetic data makes it considerably easier to create an instruction dataset mapping the expected format of similarity dataset (including queries and “hard negatives”). You’ll get here the same benefit as LLM fine-tuning: cost-saving (with a much smaller model showing the same level of performance) and appropriateness by bringing the “similarity” score closer to the expectations of your retrieval system.
+
+### 4.2 Purpose of fine-tuning: robustness
+This is the primary objective of fine-tuning in a RAG context: reinforcing the overall robustness of document synthesis. Prompt engineering has built-in limitations. Even powerful models have conflicting incentives as they are built for generalists use, obviously beyond your specific RAG system but even beyond RAG in general. LLMs are likely to hallucinate more than you’d want, simply because model providers can only restrict their creativity to a point.
+
+Robustness has several underlying implications in a RAG context.
+* **Content fidelity**: the model should generally not add more than what is presented in the retrieved document, at least beyond some general knowledge.
+* **Dealing with uncertainty**: queries can fail for a variety of reasons, either because there are no proper answers in the RAG dataset or because retrieval fails. Models should not only anticipate issues but act as a fallback when it happens, typically by stating a negative answer.
+* **Tracking the information**: RAG-dedicated models have increasing support for citation and references of each individual statement from the retrieved sources. I think the approach was originally introduced by Perplexity.
+* **Setting the stage for verifiability**: as one step further, references can be associated with quote excerpts from the retrieved sources. Beyond the potential gain of accuracy, this process makes it easier for lay users to navigate to the original sources. Just making available the complete original document turns out to be very cumbersome for fact-checking.
+
+Our RAG projects at PleIAs were conceived in the context of high constraints for verifiability (public sector, large legacy privacy companies). We took inspiration from Wikipedia. The collaborative encyclopedia had to deal with similar constraints as an expert RAG system: texts cannot be attributed to individual authors that take responsibility for publication and trustworthiness. Accuracy stems mostly from verifiability, by associating individual statements to secondary sources and easing verification.
+
```

Comment:
"easing verification" = ?

Reply:
exactly that. secondary sources + exact passage when they appear, to limit supplementary work of cross-checking.